public/friendlycode/js/friendlycode.js

```diff
@@ -77,9 +81,36 @@ define(function(require) {
       editor.panes.preview.setViewLink(info.viewURL);
     });
 
+    // Bramble: Load the project Files into the fs
+    var initFs = function(callback) {
+      if(!makeDetails || !makeDetails.title) {
+        makeDetails = ProjectFiles.generateDefaultProject();
+        ProjectFiles.load(makeDetails, { defaultTemplate: defaultContent }, callback);
+        return;
+      }
+
+      if(makeDetails.isNew) {
+        makeDetails = ProjectFiles.generateDefaultProject(makeDetails.title);
+        ProjectFiles.load(makeDetails, {
+          isNew: true,
+          defaultTemplate: defaultContent,
+          csrfToken: $("meta[name='csrf-token']").attr("content"),
+          persistenceURL: options.appUrl + "/updateProjectFile"
```

Comment:
I can't do `Path.join` because it also normalizes the result. So, `Path.join("http://localhost:2015/", "updateProjectFile")` will result in `http://localhost:2015/localhost:2015/updateProjectFile`